VirtualReality/MRML/vtkVirtualRealityViewInteractorStyle.cxx

```diff
@@ -886,34 +782,24 @@ void vtkVirtualRealityViewInteractorStyle::OnEndGesture()
 //----------------------------------------------------------------------------
 
 //----------------------------------------------------------------------------
-void vtkVirtualRealityViewInteractorStyle::MapInputToAction(
-  vtkEventDataDevice device, vtkEventDataDeviceInput input, int state)
+int vtkVirtualRealityViewInteractorStyle::GetMappedAction(vtkCommand::EventIds eid)
 {
-  if (input >= vtkEventDataDeviceInput::NumberOfInputs || state < VTKIS_NONE)
-  {
-    return;
-  }
-
-  int oldstate = this->InputMap[static_cast<int>(device)][static_cast<int>(input)];
-  if (oldstate == state)
-  {
-    return;
-  }
+  // Since both `vtkEventDataAction::Press` and `vtkEventDataAction::Release` actions are
+  // added together to the map when using the 2-argument function
+  // `MapInputToAction(vtkCommand::EventIds eid, int state)`, and we are only using this
+  // version of the function, to get the state for the corresponding `(eventId, action)`,
+  // we simply lookup for `(eventId, vtkEventDataAction::Press)` to check if an event
+  // is already mapped.
+  vtkEventDataAction action = vtkEventDataAction::Press;
 
-  this->InputMap[static_cast<int>(device)][static_cast<int>(input)] = state;
-  //this->AddTooltipForInput(device, input); //TODO:
+  decltype(this->InputMap)::key_type key(eid, action);
 
-  this->Modified();
```

Comment:
Since this is removed code and we the function being reviewed is a "getter" called `vtkVirtualRealityViewInteractorStyle::GetMappedAction`, I don't think we should call `Modified` .